wsdl/ver20/analytics/humanbody.xsd

```diff
@@ -225,18 +225,21 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 		</xs:sequence>
 		<xs:anyAttribute processContents="lax"/> 
 	</xs:complexType>
-	<xs:simpleType name="KnapsackCategory">
+	<xs:simpleType name="BagCategory">
 		<xs:restriction base="xs:string">
-			<xs:enumeration value="SingleShoulderBag"/>
+			<xs:enumeration value="ShoulderBag"/>
 			<xs:enumeration value="Backpack"/>
-			<xs:enumeration value="Other"/>
+			<xs:enumeration value="ToteBag"/>
+			<xs:enumeration value="Briefcase"/>
```

Comment:
Request to extend the enum to include 'Suitcase' under "BagCategory"?

Reply:
@HansBusch I also agree with these extensions.